pkgs/tools/security/tcplay/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchFromGithub, cmake, pkgconfig, libgcrypt, libuuid, devicemapper, libudev, openssl, libgpgerror }:
+
+stdenv.mkDerivation rec {
+  name = "tcplay-2.0";
+
+  src = fetchFromGithub {
+    owner = "bwalex";
+    repo = "tc-play";
+    rev = "v2.0";
+    sha256 = "0bg46ijjdzrfy5yv090sh6rf5wyrpdias2xiciip8qp86w7qj5qf";
+  };
+
+  buildInputs = [ cmake pkgconfig libgcrypt libuuid devicemapper libudev openssl libgpgerror ];
+
+  configurePhase = ''
```

Comment:
cmake's setup hook also defines a configure phase, are you sure it is necessary to override it?